examples/advanced/quantum_volume.ipynb

```diff
@@ -0,0 +1,127 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running the Quantum Volume Algorithm\n",
+    "This example walks through the steps of running the quantum volume algorithm on square matrices. It is intended to mirror Algorithm 1 of https://arxiv.org/pdf/1811.12926.pdf. In general, we will generate a model circuit, classically compute its Heavy Output Group, then run various samplers (currently, ideal and noisy simulators) to evaluate how often they generate Heavy results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import cirq\n",
+    "from examples.advanced import quantum_volume\n",
+    "import numpy as np\n",
+    "\n",
+    "# Some set-up variables. Feel free to play with these.\n",
+    "num_repetitions = 11 # This is supposed to be >= 100.\n",
+    "depths = range(2, 5) # The depths and number of qubits\n",
+    "\n",
+    "# Here is the important set-up: The samplers. These are what will be\n",
+    "# run on the generated circuit, and then evaluated.\n",
+    "class SamplerConfig:\n",
```

Comment:
This class is mis-named. You're mostly configuring the plot as opposed to the sampler.

The class is so small, and the context so local, that you might just want to use a NamedTuple or even just tuple. You can merge all the scatter plot stuff into a single kwargs, leaving you with `Tuple[cirq.Sampler, Dict[str, Any]]`.

Anything you can do to simplify this code, do it.

Reply:
Sounds good, I made it a simple Dict[str, Any] for now, let me know if you prefer me to separate the Sampler and the plotting config.